src/Components/TravelDetails.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import { Travels, Travel } from "../Interface/Travel";
+import useFetch from "./useFetch";
+import data from "../Data";
+import { ButtonDiv, DayDiv, FormDayDiv, StandardButton, TravelDiv } from "./Style";
+import { Form, Input, Rate } from "antd";
+import ButtonClassComponent from "./ButtonClassComponent";
+
+const { TextArea } = Input;
+
+type Days = {
+  dayNumber: string;
+  dayDesc: string;
+};
+
+const desc = ["terrible", "bad", "normal", "good", "wonderful"];
+
+const TravelDetails = () => {
+  const [value, setValue] = useState<Days>({ dayNumber: "", dayDesc: "" });
+  const [list, setList] = useState([{ dayNumber: "", dayDesc: "" }]);
+
+  const [isPending, setIsPending] = useState(false);
+  const { id } = useParams();
+  //const {data: travel, error, isPending,} = useFetch('http://localhost:8000/travels/' + id);
+  const travel = data.find((travel) => travel.id === Number(id));
+  const navigate = useNavigate();
+
+  //Deletes the whole travel
+  const handleClick = () => {
+    fetch("http://localhost:8000/travels/" + id, {
+      method: "DELETE",
+    }).then(() => {
+      navigate("/");
+    });
+  };
+
+  /*   //Should be modified to handle new day additions
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const travelDays = {
+      dayDescription,
+    };
+    console.log(travel);
+    setIsPending(true);
+
+    fetch("http://localhost:8000/travels/", {
+      method: "POST",
+      headers: { "Content-type": "application/json" },
+      body: JSON.stringify(travelDays),
+    }).then(() => {
+      console.log("new added");
+      setIsPending(false);
+    });
+  }; */
+
+  const addToList = () => {
+    let arr = list;
+    arr.push(value);
+
+    setList(arr);
+    console.log(list);
+
+    setValue({ dayNumber: "", dayDesc: "" });
```

Comment:
Next level megoldas, ha a setListben a (...prev, newItem) megoldassal pakolod be az uj erteket.